qa/L0_model_config/autofill_noplatform/ensemble/tensor_to_input_overmapped/expected_2

```diff
@@ -0,0 +1 @@
+ensemble tensor 'temp_tensor_5' is unused in ensemble 'tensor_to_input_overmapped'
```

Comment:
Why touching ensemble test? Was this removed accidentally in earlier changes?

Reply:
Very likely. This was not failing due to my changes. The error message expected itself was incorrect - i.e. the one in qa/L0_model_config/autofill_noplatform/ensemble/tensor_to_input_overmapped/expected

Reply:
There is no reason to keep expected as the error message is changed, you should just modify expected and remove the empty expected_2.

Reply:
Sure